docker/build/internal_files/entrypoint.sh

```diff
@@ -48,9 +48,16 @@ EOF
 
     export NGINX_APISERVER_ADDR=${NGINX_APISERVER_ADDRESS:-http://apiserver:8008}
     export NGINX_FILESERVER_ADDR=${NGINX_FILESERVER_ADDRESS:-http://fileserver:8081}
-
     envsubst '${NGINX_APISERVER_ADDR} ${NGINX_FILESERVER_ADDR}' < /etc/nginx/clearml.conf.template > /etc/nginx/nginx.conf
 
+    cp /usr/share/nginx/html/index.html /usr/share/nginx/html/index.html.origin
```

Comment:
Can we just move this into the `if` condition? I don't think it's used anywhere else 🙂

Reply:
Totally agree.